valhalla/baldr/graphtile.h

```diff
@@ -315,7 +315,7 @@ class GraphTile {
 
   // Graph tile memory, this must be shared so that we can put it into cache
   // Apparently you can std::move a non-copyable
-  boost::shared_array<char> graphtile_;
+  boost::shared_ptr<std::vector<char>> graphtile_;
```

Comment:
I think replacing this with `boost::scoped_ptr` will make it non-copyable. or as you said we can just use plain vector and make sure `GraphTile` itself non-copyable.